lib/posttypes/pcc-story.php

```diff
@@ -121,7 +121,7 @@ function data()
     $cmb->add_field([
         'name' => __('Storyteller', 'pcc-framework'),
         'desc' =>
-            'Name of the person who is telling this story.',
+            __('Name of the person who is telling this story. <a href="./post-new.php?post_type=pcc-person">Add a new person</a> if they do not appear on the list.'),
```

Comment:
For these, I'd prefer if you use the following pattern:

```suggestion
            sprintf( __('Name of the person who is telling this story. <a href="%s">Add a new person</a> if they do not appear on the list.'), admin_url('/post-new.php?post_type=pcc-person')),
```
Using `admin_url()` is safer for retrieving the full URL to the admin page, and `sprintf` keeps code out of the translatable string.